src/frontend/pages/api/cart.ts

```diff
@@ -49,4 +50,4 @@ const handler = async ({ method, body, query }: NextApiRequest, res: NextApiResp
   }
 };
 
-export default handler;
+export default InstrumentationMiddleware(handler);
```

Comment:
Every api route is now wrapped with this High Order Functions which provides the manual instrumentation